verl/workers/fsdp_workers.py

```diff
@@ -594,7 +594,9 @@ def _build_rollout(self, trust_remote_code=False):
         self.model_config = model_config
 
         # 2. build rollout device mesh
-        infer_tp = self.config.rollout.tensor_model_parallel_size * self.config.rollout.data_parallel_size
```

Comment:
Currently, the rollout device mesh is only used for sglang to update weights:
https://github.com/volcengine/verl/blob/main/verl/workers/rollout/sglang_rollout/sglang_rollout.py#L188-L189

There should be only rank(tp=0,pp=0,dp=0) to trigger `update_weights` via sglang http server. If we don't set `inf_tp=tp*dp`, there' will be multiple ranks: (tp=0,pp=0,dp=0), (tp=0,pp=0,dp=1), (tp=0,pp=0,dp=2), ... trigger update_weights.
https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/weight_sync/utils.py#L30